demo/.env.example

```
@@ -2,7 +2,7 @@
 GROUP_SERVICE_URL=https://atproto-group-gate-staging.up.railway.app
 GROUP_SERVICE_DID=did:web:atproto-group-gate-staging.up.railway.app
 EPDS_URL=https://epds1.test.certified.app
-SESSION_SECRET=change-me-to-a-random-string
+SESSION_SECRET=  # required: generate with e.g. openssl rand -hex 32
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)